### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by tocic**

### CODE REVIEW

File: repologyapp/templates/docs/bots.html [hunk 1]
```diff
@@ -59,7 +59,7 @@ <h4>repology-linkchecker</h4>
 <a href="https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol#Request_methods">HEAD</a>
 request first, and only if that fails it falls back to a
 <a href="https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol#Request_methods">GET</a>
-request. This means that it most cases the robot won't retrieve the contents of an URL,
```
Comment (AMDmi3) [inline repologyapp/templates/docs/bots.html:62]: I though _an_ comes before a vowel, what am I missing here? (same question for _an usable_)

Comment (tocic) [reply repologyapp/templates/docs/bots.html:62]: @AMDmi3 https://www.quora.com/Why-dont-we-use-an-before-a-hard-u-sound-like-in-Universe

**Comment by AMDmi3** [comment]
Thank you very much!

**Comment by tocic** [comment]
Btw, you could have selected the `Squash and merge` option when merging this PR. That way, there would have been [no merge commits](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/about-merge-methods-on-github#squashing-your-merge-commits) and all the `fixup: ...` and `amend: ...` commits ([created with the --fixup option](https://git-scm.com/docs/git-commit/en#Documentation/git-commit.txt---fixupamendrewordltcommitgt)) would have been automatically squashed into one commit.

**Comment by AMDmi3** [comment]
I assume that if PR contains some history, it's worth preserving it. I didn't know about fixup/amend commits though, but apparently `squash and merge` would squash not only them but self-contained separate commits as well, which I don't want. Unfortunately there's no option similar to `git rebase --autosquash`, so I can only suggest to prepare the PR the way you want to see it in the history. I'm completely OK with force-pushing to PR branches.

**Pull request merged**